src/ovp/poster.ts

```diff
@@ -22,7 +22,8 @@ function addKalturaPoster(
   const playerWidth = dimensions.width;
   const playerHeight = dimensions.height;
   if (typeof playerPoster === 'string' && THUMBNAIL_REGEX.test(playerPoster) && playerPoster === mediaConfigPoster) {
```

Comment:
should the if condition be on rawThumbnailUrl instead ?
especially since we're testing poster for being set, but then in line 25 we use rawThumbnailUrl (which we didn't test whether it's set or not)

Reply:
hmmm yeah, it's a good idea to check the `rawThumbnailUrl`- i will **add** a check for it (not as a replacement to the `playerPoster`).